src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard2.0;netstandard1.1</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard2.0;netstandard1.1;netcoreapp3.1</TargetFrameworks>

```

Comment:
netcoreapp 3.1 is out of support. 
Shall we move directly to .net 5 or .net 6?

Reply:
.net 5 is out of support, netcoreapp3.1 is going out of support in December. I can add .net 6. But I think we should keep netcoreapp3.1 support for now and maybe remove it in 8.x

Reply:
I've created a .NET 6 service that uses `ODataMessageWriter` and was able to us the new `ODataUtf8JsonWriter` and it worked as expected. So I decided not to add `net6.0` explicitly in the library for now since I'd probably also need to do that for test and other projects. Maybe that's something that can be done separately.